pkg/cmd/image/list.go

```diff
@@ -45,22 +45,42 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func List(ctx context.Context, options types.ImageListOptions) error {
+func ListCommandHandler(ctx context.Context, options types.ImageListOptions) error {
 	client, ctx, cancel, err := clientutil.NewClient(ctx, options.GOptions.Namespace, options.GOptions.Address)
 	if err != nil {
 		return err
 	}
 	defer cancel()
+	imageList, err := List(ctx, client, options.Filters, options.NameAndRefFilter)
+	if err != nil {
+		return err
+	}
+	return printImages(ctx, client, imageList, options)
```

Comment:
Given that line 49-53 (client creation) will be moved to `cmd` side, I wonder if we should just export `printImages` (i.e. -> `PrintImages`) and let `cmd.imagesAction` first call `List` below  and then call `PrintImages` to print.

I'm also okay to have the `ListCommandHandler` (naming wise, maybe just `ListHandler`, or `ListAndPrint`?) as a wrapper of the two logic (list and print).

WDYT @Zheaoli @AkihiroSuda

Reply:
> and then call PrintImages to print.

If we take this route, maybe we also want to make it consistent across all the commands (i.e., if there's anything to print, `cmd/nerdctl/X_Y.go` should call `PrintXXX` exported by `pkd/cmd/X/Y.go` to do so).

Reply:
> let cmd.imagesAction first call List below and then call PrintImages to print.

Also, if there's no single entrypoint for `cmd.imagesAction` to call, should we pass the whole `types.ImageListOptions` to both `List` and `PrintImages`? Some fields in `types.ImageListOptions` are only relevant to `List`, and some of them are only relevant to `PrintImages`.